libs/dmlf/tests/execution/basic_engine_tests.cpp

```diff
@@ -1442,7 +1442,7 @@ TEST(BasicVmEngineDmlfTests, AddFixed)
       "add", "state", "add", Params{LedgerVariant(fp64_t(4.5)), LedgerVariant(fp32_t(5.5))});
   EXPECT_TRUE(result.succeeded());
   // std::cout << result.error().message() << '\n';
-  EXPECT_EQ(result.output().As<fp64_t>(), 9.5);
+  EXPECT_EQ(result.output().As<fp64_t>(), 10.0);
```

Comment:
do you understand why this value change is necessary? (I dont)

Reply:
It's adding two numbers, fp64_t(4.5) and fp32_t(5.5). The result was -erroneously- 9.5, but it can be explained, there was no proper conversion between the fp types before, now the conversion is taking place and the numbers are converted to the proper type and added, so fp32_t(5.5) is *also* 5.5 as an fp64_t. And the result is correctly 10.0. The original test was just using a wrong value.